protocol/src/main/java/org/apache/drill/exec/proto/UserBitShared.java

```diff
@@ -13595,6 +13597,17 @@ public long getEnd() {
       return end_;
     }
 
+    public String getDuration() {
```

Comment:
The ProfileInfo class is actually where the change was made to retrieve the value from the model. We don't create any duration field, as it is computed based on the other 2 fields (start,end) and simply provide a getter for that. However, the ProfileInfo class is used only by the profiles listing freemarker template (list.ftl). 
For the individual query profile's template (profile.ftl), the model from which we retrieve the duration value is auto-generated by the protobuf, making this a lot more complicated to implement. Hence, we took the JavaScript approach. 
I'll probably refine it to strictly follow the MVC model when I'm attempting a fix for DRILL-5190 https://issues.apache.org/jira/browse/DRILL-5190  [Display planning time for a query in its profile page ] , since that would require me to modify the protobuf.